app/src/main/java/com/github/se/studybuddies/ui/groups/GroupMembers.kt

```diff
@@ -0,0 +1,281 @@
+              }
+        }
+      }
+}
+
+@Composable
+fun Name(nameState: MutableState<String>) {
+  Spacer(Modifier.height(20.dp))
+  Text(
+      nameState.value,
+      textAlign = TextAlign.Center,
+      modifier = Modifier.testTag("ShowGroupNameInGroupMember"))
+}
+
+@Composable
+fun MemberItem(
+    groupUID: String,
+    user: User,
+    navigationActions: NavigationActions,
+    db: DbRepository,
+    membersCount: Int
+) {
+  Box(
+      modifier =
+          Modifier.fillMaxWidth()
+              .background(Color.White)
+              .drawBehind {
+                val strokeWidth = 1f
+                val y = size.height - strokeWidth / 2
+                drawLine(Color.LightGray, Offset(0f, y), Offset(size.width, y), strokeWidth)
+              }
+              .testTag(user.username + "_box")) {
+        Row(modifier = Modifier.fillMaxWidth().padding(16.dp)) {
+          Box(modifier = Modifier.size(52.dp).clip(CircleShape).background(Color.Transparent)) {
+            Image(
+                painter = rememberImagePainter(user.photoUrl),
+                contentDescription = stringResource(id = R.string.user_picture),
+                modifier = Modifier.fillMaxSize(),
+                contentScale = ContentScale.Crop)
+          }
+          Spacer(modifier = Modifier.size(16.dp))
+          Text(
+              text = user.username,
+              modifier = Modifier.align(Alignment.CenterVertically),
+              style = TextStyle(fontSize = 20.sp),
+              lineHeight = 28.sp)
+          Spacer(modifier = Modifier.weight(1f))
+          MemberOptionButton(groupUID, user.uid, user.username, navigationActions, db, membersCount)
+        }
+      }
+}
+
+@Composable
+fun MemberOptionButton(
+    groupUID: String,
+    userUID: String,
+    username: String,
+    navigationActions: NavigationActions,
+    db: DbRepository,
+    membersCount: Int
+) {
+  var isRemoveUserDialogVisible by remember { mutableStateOf(false) }
+  val expandedState = remember { mutableStateOf(false) }
+  val groupViewModel = GroupViewModel(groupUID, db)
+  Row {
+    IconButton(
+        onClick = { expandedState.value = true },
+    ) {
+      Icon(
+          imageVector = Icons.Default.MoreVert,
+          tint = Blue,
+          contentDescription = stringResource(R.string.dots_menu))
+    }
+    DropdownMenu(
+        expanded = expandedState.value, onDismissRequest = { expandedState.value = false }) {
+          GROUPS_MEMBERS_DESTINATIONS.forEach { item ->
+            DropdownMenuItem(
+                onClick = {
+                  expandedState.value = false
+                  when (item.route) {
+                    Route.USERREMOVE -> {
+                      isRemoveUserDialogVisible = true
+                    }
+                  }
+                }) {
+                  Spacer(modifier = Modifier.size(16.dp))
+                  Text(item.textId)
+                }
```

Comment:
By using material3 library, you should make a small change, adding a parameter text and putting the content inside. It should look something like this:
```
DropdownMenuItem(
    onClick = {
        expandedState.value = false
        when (item.route) {
            Route.USERREMOVE -> {
                isRemoveUserDialogVisible = true
            }
         }
    },
    text = {
        Spacer(modifier = Modifier.size(16.dp))
        Text(item.textId)
    }
)
```

Reply:
Thanks, didn't now how to correct this so easily